sensor/tests/resource/role/role_test.go

```diff
@@ -69,6 +87,66 @@ func (s *RoleDependencySuite) Test_PermutationTest() {
 	)
 }
 
+func matchBinding(namespace, id string) resource.MatchResource {
+	return func(resource *central.MsgFromSensor) bool {
+		if resource.GetEvent() == nil || resource.GetEvent().GetBinding() == nil {
+			return false
+		}
+		return resource.GetEvent().GetBinding().GetId() == id && resource.GetEvent().GetBinding().GetNamespace() == namespace
+	}
+}
+
+func (s *RoleDependencySuite) Test_BindingHasNoRoleId() {
+	s.testContext.RunBare("Binding should get an update if role gets created after", func(t *testing.T, testC *resource.TestContext, _ map[string]k8s.Object) {
+		deleteDep, err := testC.ApplyFileNoObject(context.Background(), "sensor-integration", NginxDeployment)
```

Comment:
[nit] You could use the constant `DefaultNamespace` from the helper to set the namespace.